Android/Sanku_Yogesh/Beginner/ContactApp/app/src/main/java/android/example/contactapp/contact.java

```diff
@@ -0,0 +1,14 @@
+package android.example.contactapp;
+
+public class contact {
```

Comment:
Class names are always uppercase. Pls, go thru the java coding standard + typos should not be there.